src/content/docs/nrql/using-nrql/lookups.mdx

```diff
@@ -16,31 +16,45 @@ import nrqlLookupQueryTranslateItemIds from 'images/nrql_screenshot-crop_lookup-
 
 import nrqlLookupQueryGeoip from 'images/nrql_screenshot-crop_lookup-query-geoip.webp'
 
-When you [upload CSV-format lookup tables](/docs/logs/ui-data/lookup-tables-ui), you can use the `lookup()` function to access that data in your NRQL queries.
-
-## Why use lookup tables? [#why]
-
-For why you'd use lookup tables and how to upload them, see the [Lookup tables overview](/docs/logs/ui-data/lookup-tables-ui/#overview).
+When you [upload CSV-format lookup tables](/docs/logs/ui-data/lookup-tables-ui), you can use the `lookup()` function to access that data in your NRQL queries. You can use lookup tables to query data that's not in your New Relic account, help you parse telemetry data, and customize your data groupings.
 
 ## Basic query syntax [#basic-syntax]
 
-Let's say you've named your table `storeNames`. This query will select all data from that table:
+The following lookup query has the basic syntax for this function. It uses a table named `storeNames` and selects all data from that table:
```

Comment:
Or even just "This lookup query shows the basic syntax for the lookup() function."

Reply:
It's ok to repeat proper nouns if it makes things more clear :)